templates/default/client.rb.erb

```diff
@@ -68,10 +68,12 @@ rescue NameError => e
   Chef::Log.error e
 end
 
+<% if Gem::Version.new(Chef::VERSION) < Gem::Version.new('12.8') -%>
```

Comment:
I used `Gem::Version` because of a comment here: https://github.com/chef/chef/blob/05158511a3e85670f6937151a35705ff4a333053/lib/chef/version.rb#L30-L33

However, I might have misinterpreted. It sounds like in order to work with pre-release versions one should always use `Gem::Version`, which I've done here. If it makes more sense to use `Chef::Version`, I'm happy to change accordingly.